packages/plugin-vue/src/handleHotUpdate.ts

```diff
@@ -147,8 +147,10 @@ export async function handleHotUpdate(
     if (!templateModule) {
       affectedModules.add(mainModule)
     } else if (mainModule && !affectedModules.has(mainModule)) {
+      const cssLangs = `\\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)($|\\?)`
```

Comment:
can we use the `cssLangs` from `packages/vite/src/node/plugins/css.ts` instead of duplicating it?

Reply:
Okay.